docs/hardware.md

```diff
@@ -23,13 +23,9 @@ DietPi runs also on one of the first models **Raspberry Pi 1**, launched in 2012
 
 ![Raspberry Pi 1 Model B photo](assets/images/raspberry-pi-1b.jpg){: width="500" height="242" loading="lazy"}
 
-and even on the compact form **Raspberry Pi Zero W**
+and even on the compact form **Raspberry Pi Zero**, both **Zero W / WH** and **Zero 2 W**
 
-![Raspberry Pi Zero W photo](assets/images/raspberry-pi-zero-w.jpg){: width="500" height="333" loading="lazy"}
-
-Quick intro:
-
-<iframe src="https://www.youtube-nocookie.com/embed/sajBySPeYH0?rel=0" frameborder="0" allow="fullscreen" width="560" height="315" loading="lazy"></iframe>
```

Comment:
Was it intended that the YouTube video got removed?

Reply:
Maybe I was quick on removing it - my idea behind is to not promote it too much - with the launch of `Zero 2 W`, the RP4 is not the latest.

Reply:
Replaced the video, highlighting the `New Zero`

Reply:
Good idea 👍